docs/topics/db/examples/one_to_one.txt

```diff
@@ -149,3 +149,29 @@ Query the waiters::
     <QuerySet [<Waiter: Joe the waiter at Demon Dogs the restaurant>]>
     >>> Waiter.objects.filter(restaurant__place__name__startswith="Demon")
     <QuerySet [<Waiter: Joe the waiter at Demon Dogs the restaurant>]>
+
+As the relation is defined on the Restaurant side, if a place is deleted it will delete its restaurant:
+
+    >>> p4 = Place.objects.create(name='Place One', address='Address 1')
+    >>> restaurant = Restaurant.objects.create(place=p4)
+    >>> p4.delete()
+    (2, {'one_to_one.Restaurant': 1, 'one_to_one.Place': 1})
+
+A ``Place`` can also be an ``UndergroundBar``:
+
+.. code-block:: python
+
+    from django.db import models
+
+    class UndergroundBar(models.Model):
+        place = models.OneToOneField(Place, on_delete=models.SET_NULL, null=True)
```

Comment:
Sounds weird that a bar can lives without a Place 🤔 ? I took that model code from `tests/one_to_one/models.py`

Reply:
The 2nd example is more about the `on_delete` than the one-to-one relationship. I would remove it and add a reference to the `on_delete` (like in "Many-to-one relationships" docs).